packages/backend/src/open_payments/payment/outgoing/errors.ts

```diff
@@ -79,7 +83,9 @@ export const errorToMessage: {
   [OutgoingPaymentError.InactiveWalletAddress]: 'inactive wallet address',
   [OutgoingPaymentError.InvalidAmount]: 'invalid amount',
   [OutgoingPaymentError.NegativeReceiveAmount]: 'negative receive amount',
-  [OutgoingPaymentError.InvalidReceiver]: 'invalid receiver'
+  [OutgoingPaymentError.InvalidReceiver]: 'invalid receiver',
+  [OutgoingPaymentError.InvalidGrantSpentAmountState]:
+    'invalid grant spent amounts'
```

Comment:
I think we should differentiate between errors served up to an Open Payments client, or those that are just on the Admin API (since this error would not be necessarily useful to an OP client)

Reply:
> OK I think I've sidestepped this by making it more typesafe here: 934e9f6 Yay, make the invalid state unrepresentable.
Getting rid of the error entirely works great :) 

My original thinking was to separate `errorToMessage`, and have an `errorToGraphQLMessage` vs `errorToOpenPaymentsMessage` (something like this) such that we dont show "invalid grant spent amounts" to the OP client, and maybe something more generic instead. 
As an aside, we would never end up throwing the `InvalidGrantSpentAmountState` error in the Admin API since there is no grant associated when its a first-party, Admin API payment.